### DISCUSSION TITLE
Getting 'blocked by CORS policy: Method PATCH is not allowed by Access-Control-Allow-Methods in preflight response.' on React app

### DISCUSSION THREAD

**Discussion by kanpizza1150** [directus/directus#16082] [Archive]
### Describe the Bug

Getting CORS error when calling 
```
await directus.items('projects').updateOne(id,body)
```
But `.createOne`, `.readByQuery`, `.readOne`, `.readMany` are working fine.

### To Reproduce

```
const directus = new Directus(URL_ENDPOINT)
await directus.items('projects').updateOne(id, payload);
```

### Errors Shown

Access to XMLHttpRequest at 'http://xxxxxx:xxxx/items/projects/f24b12ef-2e72-483b-8774-24a2c4bbb41c' from origin 'http://localhost:3000' has been blocked by CORS policy: Method PATCH is not allowed by Access-Control-Allow-Methods in preflight response.

### What version of Directus are you using?

^10.1.4

### What version of Node.js are you using?

v16.17.0

### What database are you using?

OracleDB

### What browser are you using?

Chrome

### How are you deploying Directus?

DigitalOcean

**Comment by br41nslug** [comment]
Have you added origin `localhost:3000` to the CORS settings of your Directus instance? https://docs.directus.io/self-hosted/config-options.html#cors

**Comment by dnk8n** [comment]
I am getting similar issue for graphiql introspection against my project in Directus Cloud. How do I solve this?

**Comment by frank4177** [comment]
if you have any cors unblocking chrome extension, turn it off
